src/zope/app/pagetemplate/tests/test_talesapi.py

```diff
@@ -44,63 +44,42 @@ def sizeForDisplay(self):
     def getName(self):
         return u'apples'
 
-testObject = TestObject()
-
-def title():
-    """
-    >>> api = ZopeTalesAPI(testObject)
-    >>> api.title
-    u'apple cart'
-    """
-
-def description():
-    """
-    >>> api = ZopeTalesAPI(testObject)
-    >>> api.description
-    u'This object stores some number of apples'
-    """
-
-def name():
-    """
-    >>> api = ZopeTalesAPI(testObject)
-    >>> api.name()
-    u'apples'
-    """
-
-def title_or_name():
-    """
-    >>> api = ZopeTalesAPI(testObject)
-    >>> api.title_or_name()
-    u'apple cart'
-
-    >>> testObject = TestObject()
-    >>> testObject.title = u""
-    >>> api = ZopeTalesAPI(testObject)
-    >>> api.title_or_name()
-    u'apples'
-    """
-
-def size():
-    """
-    >>> api = ZopeTalesAPI(testObject)
-    >>> api.size()
-    u'5 apples'
-    """
-
-def modified():
-    """
-    >>> api = ZopeTalesAPI(testObject)
-    >>> api.modified
-    datetime.datetime(2003, 1, 2, 3, 4, 5)
-    """
-
-def created():
-    """
-    >>> api = ZopeTalesAPI(testObject)
-    >>> api.created
-    datetime.datetime(2000, 10, 1, 23, 11)
-    """
```

Comment:
There were few enough doctests here that I was able to get away with not using a `RENormalizer` and just convert to unit tests 😄